ci/build-test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+# Builds and tests Python package
+
+set -ue
+
+pip install build
+
+python -m build .
+
+for PKG in dist/*; do
+  echo "$PKG"
+  pip uninstall -y rapids-metadata
+  pip install "$PKG[test]"
```

Comment:
```suggestion
  pip install --force-reinstall "$PKG[test]"
```

You could avoid the need to `pip uninstall` (and the corresponding log line with a warning about this project not already being installed) by passing `--force-reinstall` to `pip install`.

From `pip install --help`.

```text
--force-reinstall           Reinstall all packages even if they are already up-to-date.
```